web-app/src/main/java/com/google/sps/servlets/AnalyticsServlet.java

```diff
@@ -61,7 +61,7 @@ public void init() {
 
   private boolean isReportNearLine(PoliceReport report, Coordinates[] waypoints) {
     Coordinates reportLocation = new Coordinates(report.getLat(), report.getLng());
-    for (int index = 0; index < waypoints.length - 1; i++) {
+    for (int index = 0; index < waypoints.length - 1; index++) {
```

Comment:
nit: "i" is probably fine for the variable name, it's become so standard that it's basically a word